tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/Dictionary/Dictionary.spec.ts

```diff
@@ -2,78 +2,117 @@ import {test} from '@umbraco/playwright-testhelpers';
 import {expect} from "@playwright/test";
 
```

Comment:
```suggestion
import * as crypto from 'crypto';
```
When I ran the API tests for dictionary I got the error ReferenceError: crypto is not defined. So we need to define the import for the test to work